### PR TITLE
Accessibility checks

### PR DESCRIPTION
**Pull request by rugwizangoga**
# I found the following during my Accessibility checks:

- Page title  (no issues found)
- Image text alternatives** (img alt="alt text")  (renamed alt texts on imgs from v to Angelistic here https://github.com/rugwizangoga/Portifolio/blob/c1991cb1f5d1f455c65e823cfae0d9782096ca0b/index.html#L58 and here https://github.com/rugwizangoga/Portifolio/blob/c1991cb1f5d1f455c65e823cfae0d9782096ca0b/index.html#L186 )
- Text headings (no issues found)
- Color contrast (no issues found)
- Resize   (no issues found)
- Interaction (added labels in index.html https://github.com/rugwizangoga/Portifolio/blob/e66933c958fc5dbc1918b9cd0a7aaaab9d583791/index.html#L227-L233 and made them invisible in style.css https://github.com/rugwizangoga/Portifolio/blob/e66933c958fc5dbc1918b9cd0a7aaaab9d583791/style.css#L40-L42

- Moving content (no issues found)
- Multimedia (no issues found because the project didn't require a video or audio of any kind)
- The basic structure of the page (I moved the main tag from above the header to below the header https://github.com/rugwizangoga/Portifolio/blob/e66933c958fc5dbc1918b9cd0a7aaaab9d583791/index.html#L11-L24  and I added meta tags with description, keywords, and author attributes  https://github.com/rugwizangoga/Portifolio/blob/c1991cb1f5d1f455c65e823cfae0d9782096ca0b/index.html#L5-L7

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,7 +52,7 @@ <h2 class="connect">LET’S CONNECT</h2>
                         <a href="https://github.com/rugwizangoga"><img src="./images/github.svg" alt="github"></a>
                     </li>
                     <li>
-                        <a href=""><img src="./images/v.svg" alt="v"></a>
+                        <a href=""><img src="./images/v.svg" alt="Angelistic"></a>
                     </li>
                 </ul>
             </div>
```
Comment (aimemalaika) [inline index.html:61]: - [ ] Kindly add an URL to all your links

File: index.html [hunk 2]
```diff
@@ -8,20 +8,20 @@
     <link rel="stylesheet" href="./style.css">
```
Comment (aimemalaika) [inline index.html:11]: - [ ] Kindly add a meta tag with description attribute to the head of the page

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️

- [ ] Your accessibility score
Screen Shot 2022-08-30 at 16 59 34

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @rugwizangoga, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**